packages/editor/src/components/UiBlockHeader.css

```diff
@@ -6,3 +6,38 @@
 .canvas[data-help-paddings='false'] .header-block__visible {
   display: none;
 }
+
+.canvas .badge-label {
+  background: transparent;
+  border: none;
+  color: inherit;
+  min-height: auto;
+  width: auto;
+  padding: 0;
+  line-height: inherit;
+  font-size: inherit;
+  display: flex;
+}
+
+.canvas .badge-label:empty::after {
+  content: '.';
+  visibility: hidden;
+}
+
+.canvas .badge-label .ui-flex {
+  padding: 0;
+  margin: 0 3px 0 3px;
```

Comment:
?
why not on the .badge-label?
why not `margin-inline: 3px;`

Reply:
the margin need to be on the ui-flex as within the output element